packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -94,18 +69,22 @@ export const _CaptionsBanner = (props: _CaptionsBannerProps): JSX.Element => {
           ) : (
             <Persona {...personaOptions} />
           );
+          // if display name is too long, replace with ...
+          if (caption.displayName.length > 15) {
+            caption.displayName = `${caption.displayName.substring(0, 16)}...`;
```

Comment:
Should this account for the ... in the length as well?
Perhaps we need to substring(0,14) or substring(0,15) to account for the ellipsis

Reply:
Yeah I'm keeping the length to be 15 charaters + ... according to the design